specification.markdown

```diff
@@ -1,100 +1,100 @@
 # Introduction
 
-`EBML`, short for Extensible Binary Meta Language, specifies a binary and octet (byte) aligned format inspired by the principle of XML (a framework for structuring data).
+EBML, short for Extensible Binary Meta Language, specifies a binary and octet (byte) aligned format inspired by the principle of XML (a framework for structuring data).
```

Comment:
Should the first occurrence not be «quoted», as definition?

Reply:
Fixed in the next commits

Reply:
In the `Notation and Conventions` section, not the main text.